composer.json

```diff
@@ -16,6 +16,7 @@
         "ext-mbstring": "*",
         "ext-tokenizer": "*",
         "composer/composer": "^1.7",
+        "illuminate/support": "^5.6|^6.0",
```

Comment:
Not sure about version, tell me if it's needed to update at least to 5.8. I added 6.0 for the future :wink: